src/compliance/analyser_interface.py

```diff
@@ -0,0 +1,10 @@
+from abc import ABC, abstractmethod
+from src.data.audit import Audit
+from src.data.notification import Notification
+from typing import Set
+
+
+class AnalyserInterface(ABC):
+    @abstractmethod
+    def analyse(self, audit: Audit) -> Set[Notification]:
+        pass
```

Comment:
If we replace `pass` with a comment, then this line won't be considered an uncovered line in the test coverage report. Consequently we can delete tests/compliance/test_analyser_interface.py

```suggestion
        """
        :param audit: an audit report
        :return: set of notifications for findings in the audit report
        """
```